_maps/map_files/generic/centcomm.dmm

```diff
@@ -424,9 +424,7 @@
 /turf/simulated/floor/plasteel/freezer,
 /area/ninja/holding)
 "bO" = (
-/obj/structure/closet/secure_closet/bar{
-	req_access = list(25)
-	},
+/obj/structure/closet/secure_closet/bar,
```

Comment:
Whats going on here? Seems unrelated/unintentional?
Messing with this cabinet for reference
![image](https://github.com/user-attachments/assets/cada2d77-d4a6-4d13-bfd9-a0c01032a766)

removing bar access for double reference.
![image](https://github.com/user-attachments/assets/ffe1f032-58a8-4c34-9b30-f67a797abfe5)

Reply:
SDMM sanitizing variables